examples/multi-cluster-fleet/python/agent_sandbox_fleet/cli.py

```
@@ -0,0 +1,503 @@
+def _positive_float(raw: str) -> float:
+  """argparse type for intervals. Rejects <= 0, nan and inf.
+
+  A non-positive --loop-interval makes the apply loop spin: the sleep
+  computes max(0.0, interval - elapsed), so every cycle sleeps 0 and the
+  planner re-plans and rewrites assignments.json as fast as GCS will answer.
+  nan fails the > 0 comparison here, which is the intent.
+  """
+  try:
+    val = float(raw)
+  except (ValueError, OverflowError):
+    raise argparse.ArgumentTypeError(f"{raw!r} is not a number")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)